src/main/java/io/airlift/units/DataSize.java

```diff
@@ -299,7 +299,7 @@ public int hashCode()
     {
         //This order is important, it should be in increasing magnitude.
         BYTE(1L, "B"),
-        KILOBYTE(1L << 10, "kB"),
+        KILOBYTE(1L << 10, "KB"),
```

Comment:
lowercase was correct, see https://en.wikipedia.org/wiki/Metric_prefix 

( the "more correct" would be to use "KiB", "MiB"  etc but that's a bigger change, as it would need to be backwards compat )

Reply:
SI standard is used in the source code.
https://en.wikipedia.org/wiki/Kilobyte 

> The International System of Units (SI) defines the prefix kilo as 1000 (103); per this definition, one kilobyte is 1000 bytes.[1] The internationally recommended unit symbol for the kilobyte is kB.

It might be better to use the IEC standard